Source/BluetoothError.swift

```diff
@@ -134,6 +141,8 @@ extension BluetoothError: Equatable {}
 
 public func == (lhs: BluetoothError, rhs: BluetoothError) -> Bool {
     switch (lhs, rhs) {
+    case (.ongoingScan, .ongoingScan): return true
```

Comment:
I'm not sure about name of this case. Maybe `scanInProgress` might be better?

Reply:
ok, will do 👍